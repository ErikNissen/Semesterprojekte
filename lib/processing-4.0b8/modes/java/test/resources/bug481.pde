version https://git-lfs.github.com/spec/v1
oid sha256:2d2f1a4129e588bc7c3dafc41a5b3ed059399a47ac090524a33f37db99a7da79
size 67
