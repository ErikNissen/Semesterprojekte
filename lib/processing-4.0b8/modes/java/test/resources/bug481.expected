version https://git-lfs.github.com/spec/v1
oid sha256:ea8d8b41685643d84d9f680df39b4b9fbfabd27a3c53b0fcb01e6608d4e3cc2b
size 761
